src/current/cockroachcloud/serverless-resource-usage.md

```diff
@@ -114,6 +114,7 @@ The following recommendations can help reduce the RU cost of a query by reducing
 - Use range `UPDATE` and `DELETE` statements to affect many rows in a single statement, rather than sending a separate statement per row.
 - Avoid returning columns that your application does not need.
 - Don't disable automatic statistics, as they are needed to power the [optimizer](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/cost-based-optimizer).
+- For multi-region clusters, avoid cross-region reads by using features such as [global tables](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/global-tables.md), [regional by row tables](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/regional-tables.md), and [follower reads](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/follower-reads.md) where possible.
```

Comment:
Is it worth linking to the section in plan-your-serverless-cluster.md for more details?